modules/mc/hm.nix

```
@@ -0,0 +1,161 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  mcPackage = pkgs.mc;
+  colors = config.lib.stylix.colors.withHashtag;
+in
+{
+  config = mkIf (builtins.elem mcPackage config.home.packages) {
```

Since `mcPackage` is used only once, it might be better to inline its value:

```suggestion
  colors = config.lib.stylix.colors.withHashtag;
in
{
  config = mkIf (builtins.elem pkgs.mc config.home.packages) {
```

---

Resolved by https://github.com/danth/stylix/pull/463/commits/ae1ee6efb818d63a2c0b7abb49e3250bfb2430fb.